test/Libraries/RevitNodesTests/Elements/ElementTests.cs

```diff
@@ -260,6 +260,32 @@ public void ElementCurveReferences_ExtractsExpectedReferences()
 
         #endregion
 
+        private static void GetHostedElements(Element elem)
+        {
+            var hostedElementsIncludeNothing = elem.GetHostedElements();
+            var hostedElementsIncludeOpenings = elem.GetHostedElements(true);
+            var hostedElementsIncludeEmbeddedWalls = elem.GetHostedElements(false, false, true, false);
+            var hostedElementsIncludeOpeningsAndEmbeddedWalls = elem.GetHostedElements(true, false, true, false);
+
+            //Assert right amount
+            Assert.AreEqual(3, hostedElementsIncludeNothing.Length);
+            Assert.AreEqual(4, hostedElementsIncludeOpenings.Length);
+            Assert.AreEqual(4, hostedElementsIncludeEmbeddedWalls.Length);
+            Assert.AreEqual(5, hostedElementsIncludeOpeningsAndEmbeddedWalls.Length);
+
+            //Assert right elements
+            string[] expected = new[] { "600 x 3100", "600 x 3100", "600 x 3100" };
+            CollectionAssert.AreEqual(expected, hostedElementsIncludeNothing.Select(x => x.Name)
+                                                                            .ToArray());
+        }
+
+        [Test]
+        [TestModel(@".\element.rvt")]
+        public void CanSuccessfullyGetHostedElements()
```

Comment:
The other PRs seemed to not include `Successfully` - is there anything different here ? if not, let's be consistent
```suggestion
        public void CanGetHostedElements()
```

Reply:
the other unit test should have `Successfully`, following the naming in the existing tests